src/SupabaseClient.ts

```diff
@@ -254,8 +255,8 @@ export default class SupabaseClient {
     })
   }
 
-  private _getAuthHeaders(): { [key: string]: string } {
-    const headers: { [key: string]: string } = this.headers
+  private _getAuthHeaders(): GenericObject {
+    const headers: GenericObject = DEFAULT_HEADERS
```

Comment:
@ftonato i think the merge commit (https://github.com/supabase/supabase-js/pull/258/commits/19647931b45b6ee8a16c62a7608de826b18e91a6) messed this up:

```suggestion
    const headers: GenericObject = this.headers
```